pkgs/development/python-modules/pyplatec/default.nix

```
@@ -13,11 +13,14 @@ buildPythonPackage rec {
     sha256 = "0kqx33flcrrlipccmqs78d14pj5749bp85b6k5fgaq2c7yzz02jg";
   };
 
+  patches = [
+    ./flags.patch
+  ];
```

```suggestion
  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isGNU "-std=c++11";
```
How about using `NIX_CFLAGS_COMPILE`?

---

I copied the way the old versions did it, not sure why they removed the flag, but env.NIX_CFLAGS_COMPILE works too so I changed the fix. But the build failure doesn't seem to be GCC specific (it breaks C++17 rules) so I didn't make it optional.